go/vt/vttablet/tabletserver/vstreamer/tablestreamer.go

```diff
@@ -117,12 +117,16 @@ func (ts *tableStreamer) Stream() error {
 		return err
 	}
 
-	rs, err := conn.ExecuteFetch("show tables", -1, true)
+	rs, err := conn.ExecuteFetch("show full tables", -1, true)
 	if err != nil {
 		return err
 	}
 	for _, row := range rs.Rows {
 		tableName := row[0].ToString()
+		tableType := row[1].ToString()
```

Comment:
I think that we should either switch to using named rows or add a quick len(row) check just to be sure we don't crash with index out of bounds in some odd cases.